src/ScriptBuilder/Saga/Writers/OracleCorrelationPropertyTypeConverter.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace NServiceBus.Persistence.Sql.ScriptBuilder
+{
+    /// <summary>
+    /// Not for public use.
+    /// </summary>
+    public static class OracleCorrelationPropertyTypeConverter
+    {
+        public static string GetColumnType(CorrelationPropertyType propertyType)
+        {
+            switch (propertyType)
+            {
+                case CorrelationPropertyType.DateTime:
+                    return "TIMESTAMP";
+                case CorrelationPropertyType.String:
+                    return "NVARCHAR2(200)";
+                case CorrelationPropertyType.Int:
+                    return "NUMBER(19)";
+                case CorrelationPropertyType.Guid:
+                    return "VARCHAR2(38)";
```

Comment:
Same as MySQL, I believe it's intended to allow for { } braces before and after the guid. Will be documented by the Oracle Scripts page.